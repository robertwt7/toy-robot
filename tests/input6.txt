PLACE 1,2,EAST
MOVE
MOVE
LEFT
MOVE
MOVE
MOVE
MOVE
MOVE
RIGHT
MOVE
RIGHT
MOVE
REPORT