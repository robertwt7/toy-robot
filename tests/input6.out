Move Invalid
Move Invalid
Move Invalid
4,3,SOUTH